### ISSUE TITLE
runSymfonyConsoleCommand does not work if the command has arguments

### ISSUE THREAD

**Issue by burned42** [Codeception/module-symfony#1]
In the changelog I noticed that @wirwolf recently added `runSymfonyConsoleCommand` and I tried to use it with a command that has an argument.

The problem is that when symfony determines the command name to use, it calls `getFirstArgument` on the `$input` (https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Application.php#L980) which in case of `ArrayInput` (which is used in `runSymfonyConsoleCommand`) gets you the first element of the 'parameters' array which has a key that does not start with an '-' (https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Input/ArrayInput.php#L40).

Since `runSymfonyConsoleCommand` adds the 'command' to the end of the parameters array, this only works if your command doesn't have arguments.

Note: arguments != options, see https://symfony.com/doc/current/console/input.html

Current Behavior
=============
When running
```php
$I->runSymfonyConsoleCommand(
    'app:example-command',
    ['example_argument' => 'this_is_the_argument_value'],
);
```
This is the output:
```
1) ExampleCommandCest: Try to test
 Test  tests/functional/Command/ExampleCommandCest.php:tryToTest
 Step  Run symfony console command
"app:example-command",{"example_argument":"this_is_the_argument_value"}
 Fail  Exit code in app:example-command is not equal 0 :

  Command "this_is_the_argument_value" is not defined.



Failed asserting that 1 matches expected 0.
```

Workaround
=========
Adding the 'command' manually at the start of the input array solves this issue for me.
```php
$I->runSymfonyConsoleCommand(
    'app:example-command',
    [
        'command' => 'app:example-command',
        'example_argument' => 'this_is_the_argument_value'
    ],
);
```
Because then it just gets overridden again in https://github.com/Codeception/module-symfony/blob/master/src/Codeception/Module/Symfony.php#L570

---

Am I just using this wrong, or is this actually a bug? If it's a bug, I'd be happy to provide a PR to solve it.

**Comment by burned42** [comment]
Since in the meantime I also needed to pass some console inputs to answer interactive questions (https://symfony.com/doc/current/components/console/helpers/questionhelper.html#testing-a-command-that-expects-input) I came up with the following solution, which works with arguments, options and also console input, based on the symfony documentation (https://symfony.com/doc/current/console.html#testing-commands):
```php
/**
 * @param string $command          The console command to execute
 * @param array  $parameters       Parameters (arguments and options) to pass to the command
 * @param array  $consoleInputs    Console inputs (e.g. used for interactive questions)
 * @param int    $expectedExitCode The expected exit code of the command
 *
 * @return string Returns the console output of the command
 */
public function runConsoleCommand(
    string $command,
    array $parameters = [],
    array $consoleInputs = [],
    int $expectedExitCode = 0
): string {
    $kernel = $this->grabService('kernel');
    $application = new Application($kernel);
    $consoleCommand = $application->find($command);
    $commandTester = new CommandTester($consoleCommand);
    $commandTester->setInputs($consoleInputs);

    $parameters = ['command' => $command] + $parameters;
    $exitCode = $commandTester->execute($parameters);
    $output = $commandTester->getDisplay();

    $this->assertEquals(
        $expectedExitCode,
        $exitCode,
        'Command did not exit with code '.$expectedExitCode
        .' but with '.$exitCode.': '.$output,
    );

    return $output;
}
```

This works at least with Symfony 4, I haven't checked if it works with older versions yet.

**Comment by vedro-compota** [comment]
I see same problem

**Comment by Naktibalda** [comment]
Please fix it and raise pull request.

**Comment by eortiz-tracktik** [comment]
@Naktibalda I have open a PR with the code suggested by @burned42 https://github.com/Codeception/module-symfony/pull/7

**Comment by Naktibalda** [comment]
Thanks @eortiz-tracktik 

I released it as 1.1.0
